CONTRIBUTING.md

```diff
@@ -55,13 +55,14 @@ This is generally not necessary, but in case you need to manually publish a vers
 2. `git pull`
 3. Make sure you have a clean working tree (`git status` should show no changes)
 4. `git checkout -b release-X.Y.Z` - Create a new release branch, where `X.Y.Z` is the version number you're about to release.
-5. `npm version [major | minor | patch]` - This will bump the version number in `package.json` and `package-lock.json`. e.g., `npm version minor` to bump from `1.1.0` to `1.2.0`.
-6. `git push` your branch.
-7. Make a PR, get it approved, and merge your changes to `v-next`.
-8. `git checkout v-next`
-9. `git pull`
-10. Make sure you have a clean working tree (`git status` should show no changes)
-11. `npm publish --access public` - This will automatically install and compile everything, run linting, and publish
+5. Reinstall dependencies and run build: `npm ci && npm run preprocess && npm run build`
```

Comment:
1. I think this actually needs to come before running `npm publish`, because some time may pass before the PR gets merged, and other changes might happen, so they should build directly before deploying.

2. It's too late for this commit, but could you change all these numbers to `1.` to make future git diffs easier to read? Markdown will use the correct number regardless of what we put here.

Reply:
✅